python/config_settings/BUILD.bazel

```diff
@@ -52,3 +53,12 @@ string_flag(
     # NOTE: Only public because its an implicit dependency
     visibility = ["//visibility:public"],
 )
+
+string_flag(
+    name = "bootstrap_impl",
```

Comment:
I was curious if we can find a better name that does not talk about stages. In some sense a casual user only cares if they need system python or not, so maybe that should be the flags name.

Reply:
the changelog and pr descriptions are also needs to be modified